src/_locales/en/messages.json

```diff
@@ -255,6 +255,18 @@
         "message": "$1 regex/bulk edits",
         "description": "'Regex/Bulk edits' header"
     },
+    "optionsBlockedTags": {
+        "message": "Blocked tags",
+        "description": "'Blocked tags' header"
+    },
+    "optionsBlockedTagsDesc": {
+        "message": "This extension allows you to block certain artists/albums/tracks from scrobbling.",
```

Comment:
This copy seems inconsistent with the others on the edit page - do you think we should align them?